src/main/java/org/opendatakit/aggregate/client/popups/PurgeUpToDatePopup.java

```diff
@@ -70,7 +70,7 @@ public void onValueChange(ValueChangeEvent<Date> event) {
     layout.setWidget(0, 0, new HTML(new SafeHtmlBuilder()
         .appendEscaped("Purge submissions data for:")
         .appendHtmlConstant("<br/>")
-        .appendEscaped(formSummary.getTitle() + " [" + formSummary.getId() + "]")
+        .appendHtmlConstant("<b>" + summary.getTitle() + " [" + summary.getId() + "]</b>")
```

Comment:
Why did this change from formSummary to summary?

Reply:
Never mind. Looked at the full source and it makes sense now 👍

Reply:
I copied and pasted the line from `ConfirmPurgeUpToDatePopup.java` and it worked the same because line 48 `this.summary = formSummary;` but I've reverted that to make the change smaller.